packages/collector/test/tracing/cloud/aws-sdk/v3/dynamodb/test.js

```diff
@@ -0,0 +1,227 @@
+}
+
+let mochaSuiteFn;
+
+const operationsInfo = {
+  createTable: 'create',
+  listTables: 'list',
+  scan: 'scan',
+  query: 'query',
+  getItem: 'get',
+  deleteItem: 'delete',
+  putItem: 'put',
+  updateItem: 'update'
+};
+
+const withErrorOptions = [false, true];
+
+const requestMethods = ['v3', 'v2', 'cb'];
+const availableOperations = [
+  'createTable',
+  'putItem',
+  'listTables',
+  'updateItem',
+  'getItem',
+  'scan',
+  'query',
+  'deleteItem'
+];
+
+const getNextCallMethod = require('@instana/core/test/test_util/circular_list').getCircularList(requestMethods);
+
+if (!supportedVersion(process.versions.node) || semver.lt(process.versions.node, '10.0.0')) {
+  mochaSuiteFn = describe.skip;
+} else {
+  mochaSuiteFn = describe;
+}
+
+const retryTime = config.getTestTimeout() * 2;
+
+mochaSuiteFn('tracing/cloud/aws-sdk/v3/dynamodb', function () {
+  this.timeout(config.getTestTimeout() * 3);
+
+  globalAgent.setUpCleanUpHooks();
+  const agentControls = globalAgent.instance;
+
+  describe('tracing enabled, no suppression', function () {
+    before(() => {
+      return cleanup(tableName);
+    });
+
+    const appControls = new ProcessControls({
+      appPath: path.join(__dirname, 'app'),
+      port: 3215,
+      useGlobalAgent: true,
+      env: {
+        AWS_DYNAMODB_TABLE_NAME: tableName
+      }
+    });
+
+    ProcessControls.setUpHooksWithRetryTime(retryTime, appControls);
+    withErrorOptions.forEach(withError => {
+      if (withError) {
+        describe(`getting result with error: ${withError ? 'yes' : 'no'}`, () => {
+          it(`should instrument ${availableOperations.join(', ')} with error`, () => {
+            return promisifyNonSequentialCases(verify, availableOperations, appControls, withError, getNextCallMethod);
+          });
+        });
+      } else {
+        describe(`getting result with error: ${withError ? 'yes' : 'no'}`, () => {
+          availableOperations.forEach(operation => {
+            const requestMethod = getNextCallMethod();
+            it(`operation: ${operation}/${requestMethod}`, async () => {
+              const withErrorOption = withError ? '?withError=1' : '';
+              const apiPath = `/${operation}/${requestMethod}`;
+
+              const response = await appControls.sendRequest({
+                method: 'GET',
+                path: `${apiPath}${withErrorOption}`,
+                simple: withError === false
+              });
+
+              /**
+               * Table takes some time to be available, even though the callback gives a success message
+               */
+              if (operation === 'createTable') {
+                await checkTableExistence(tableName, true);
+              }
+              return verify(appControls, response, apiPath, operation, withError);
+            });
+          });
+        });
+      }
+    });
+
+    function verify(controls, response, apiPath, operation, withError) {
```

Comment:
Looks like you do nothing with the `response` here. Should we add a check for it? I find that checking the response is pretty helpful to verify that our instrumentation code does not interfere with the result of the traced operation.